README.md

```diff
@@ -18,4 +18,4 @@ Displays the tasks and assignments that are ongoing.
 # Website
 
 - Status: https://status.realdevsquad.com/
-- Homepage: https://www.realdevsquad.com/
+- Homepage: https://www.realdevsquad.com/
```

Comment:
What changed here?

Reply:
Nothing. I tried figuring out why it is showing it changed but couldn't.
I tried to normalize line endings but even that didn't work